agent/ecscni/plugin.go

```diff
@@ -96,6 +99,8 @@ func (client *cniClient) CleanupNS(cfg *Config) error {
 	}
 
 	seelog.Debugf("Starting clean up the container namespace: %s", cfg.ContainerID)
+	os.Setenv("ECS_CNI_LOGLEVEL", logger.GetLevel())
+	defer os.Unsetenv("ECS_CNI_LOGLEVEL")
```

Comment:
May be. In a world where this variable would be configured dynamically, having this here helps. I'd like to keep it this way as it's not an expensive call and requires no modifications in future when we support dynamic config of agent's debug level